src/pages/yunkyung/Login/Login.js

```diff
@@ -1,8 +1,91 @@
 import React from 'react';
+import './Login.scss';
+
+class Login extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      userId: '',
+      userPw: '',
+      btnControl: 'disabled',
+    };
+  }
+
+  handleChange = e => {
+    const userId = e.target.name;
+    const userPw = e.target.value;
+    this.setState({
+      [userId]: userPw,
+    });
+  };
```

Comment:
- 여기도 구조분해할당 하면 좋을 듯 합니다!
```suggestion
  handleChange = e => {
    const { name, value } = e.target;
    this.setState({
      [name]: value,
    });
  };
```

Reply:
변경 완료! 호호 🤗